supporting_content/columns/commitmentdiscountunit.md

```diff
@@ -0,0 +1,23 @@
+# Column: CommitmentDiscountUnit
+
+## Example provider mappings
+
+Current column mappings found in available data sets:
+
+| Provider | Data set | Column |
+|----------|----------|--------|
+| AWS | CUR | Not available |
+| Google Cloud | BigQuery Billing Export | Not available |
+| Microsoft | Cost Details | Not available |
```

Comment:
Technically, usage commitments have UnitOfMeasure and spend commitments have BillingCurrency, right?
```suggestion
| Microsoft | Cost Details | Usage commitments: UnitOfMeasure<br>Spend commitments: BillingCurrency |
```